twa

```diff
@@ -43,10 +43,14 @@ function ensure {
     || die "Failed to run '$*'. Aborting."
 }
 
-function fetch {
+function fetch_secure {
```

Comment:
Instead of a new `fetch_secure` function, maybe just leave `fetch` as secure by default and make it insecure by calling it with `--insecure`?